query-engine/connectors/query-connector/src/interface.rs

```diff
@@ -116,11 +116,9 @@ impl AggregationSelection {
     /// Returns (<field db name>, TypeIdentifier, FieldArity)
     pub fn identifiers(&self) -> Vec<(String, TypeIdentifier, FieldArity)> {
         match self {
-            AggregationSelection::Field(field) => vec![(
-                field.db_name().to_owned(),
-                field.type_identifier.clone(),
-                field.arity,
-            )],
+            AggregationSelection::Field(field) => {
+                vec![(field.db_name().to_owned(), field.type_identifier.clone(), field.arity)]
+            }
```

Comment:
Apparently we merged with failing cargo fmt. Not cool. It was my PR, I don't know how that happened.